### ISSUE TITLE
Fix Javascript/Typescript errors

### ISSUE THREAD

**Issue by huhn511** [open-marketplace-applications/dshop#37]
- [ ] Resolve "tsconfig.json" errors

**Comment by sascha1337** [comment]
To be honest, using babel for non-browser node.js applications, is not needed anymore in 2021.
Checking the line history on git, the babel stuff is some dusty fragment from 4 years ago. 
Would love to fix that one, replace it with default v10+ node, properly enable ESLint with .eslintrc file and some useful plugins for that. In combination with the VS code extension `ErrorLens` ~ you will see inline warnings, if you are writing some code with bad practices or old | deprecated syntax, that should be changed.

**Comment by huhn511** [comment]
We uses a [SvelteKit](https://kit.svelte.dev/) preconfigured setup. Do we still need this?

**Comment by sascha1337** [comment]
Yeah, on that side, its all good. But its about those root level package.json modules that are used to spin up the backend. These days, babel is deprecated, didn't see this in-use with node.js since ages. Frontend-wise, as you correctly said, svelte takes care of these things already.  

https://github.com/open-marketplace-applications/dshop/blame/91a1e1be65cace6c57d96a39726af0bf5c9736a3/package.json#L8

![Screenshot 2021-05-15 at 22 52 01](https://user-images.githubusercontent.com/12372821/118365643-3a3f8b00-b5d0-11eb-936a-bcb3791b8f64.png)
